CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where the library sort order was lost. [#6091](https://github.com/JabRef/jabref/issues/6091)
 - We fixed an issue where brackets in regular expressions were not working. [6469](https://github.com/JabRef/jabref/pull/6469)
 - We fixed an issue where LaTeX citations for specific commands (\autocites) of biblatex-mla were not recognized. [#6476](https://github.com/JabRef/jabref/issues/6476)
+- Now the context menu of the column "Linked identifiers" of the main table gets truncated, if too long, otherwise display issues arise. [#6499](https://github.com/JabRef/jabref/issues/6499)
```

Comment:
Please keep a more 'formal' writing style. Also this entry should probably go to the changes section, as this is an enhancement which was not filed as a bug report.